src/components/Person.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export function Person({ person }) {
+  let partnerText;
+
+  if (person.isMarried) {
+    partnerText = (person.sex === 'f')
+      ? `My husband's name is ${person.partnerName}`
+      : `My wife's name is ${person.partnerName}`;
+  } else {
+    partnerText = 'I am not married';
+  }
+
+  return (
+    <>
+      <section className="Person">
+        <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+        {person.age
+          ? (<p className="Person__age">{`I am ${person.age}`}</p>) : null
+        }
```

Comment:
You can use `&&` operator here
```suggestion
        {person.age && (
          <p className="Person__age">
            {`I am ${person.age}`}
          </p>) 
        }
```